scanner/src/funTest/kotlin/storages/AbstractStorageTest.kt

```diff
@@ -139,8 +141,7 @@ abstract class AbstractStorageTest : StringSpec() {
 
             result shouldBe ADD_RESULT_SUCCESS
             storedResults.id shouldBe id
-            storedResults.results.size shouldBe 1
-            storedResults.results[0] shouldBe scanResult
+            storedResults.results should containExactly(scanResult)
```

Comment:
I think in other places we use already the infix fun `shouldContainExactly` instead. How about using it here as well?

Reply:
Hm or maybe the other way round, I like this one more. How about you?

Reply:
I think `should containExactly` read a bit better. Let's talk on Monday on which of both we want to align.

Reply:
Agree. I think you can merge this.